daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -349,7 +349,7 @@ private[lf] final class Compiler(
   private[this] def compileModule(
       pkgId: PackageId,
       module: Module,
-      preds: List[PackageId],
+      predPids: List[PackageId],
```

Comment:
```suggestion
      predPkgIds: List[PackageId],
```
We never used pids for Package ID, so I think it is better to use either PkgId` or `PackageId`